homeassistant/components/alexa/__init__.py

```diff
@@ -35,6 +35,12 @@
 CONF_FLASH_BRIEFINGS = "flash_briefings"
 CONF_SMART_HOME = "smart_home"
 DEFAULT_LOCALE = "en-US"
+VALID_ENDPOINTS = [
+    "https://api.amazonalexa.com/v3/events",
+    "https://api.eu.amazonalexa.com/v3/events",
+    "https://api.fe.amazonalexa.com/v3/events",
+]
+
```

Comment:
Should we add a comment with the web reference?

https://developer.amazon.com/en-US/docs/alexa/smarthome/send-events.html

Reply:
Good suggestion, will do 👍